### PR TITLE
msys2-runetime -  2.8Update to latest newlib version

### PR DESCRIPTION
**Pull request by JPeterMugaas**
less - 487 - Update to latest version

### CODE REVIEW

**Comment by mati865** [comment]
`error: target not found: mingw-w64-cross-libmangle` legit error.

File: mingw-w64-cross-tools-git/PKGBUILD [hunk 1]
```diff
@@ -1,60 +1,61 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
-# Contributor: Martell Malone <martellmalone@gmail.com>
 
-_realname=tools
+_realname=winpthreads
```
Comment (Alexpux) [inline mingw-w64-cross-tools-git/PKGBUILD]: Why you rename package?

File: mingw-w64-cross-tools-git/PKGBUILD [hunk 2]
```diff
@@ -1,60 +1,61 @@
...
-pkgname=("${_mingw_suff}-${_realname}-git")
+pkgname="${_mingw_suff}-${_realname}-git"
 replaces=("${_mingw_suff}-${_realname}")
 provides=("${_mingw_suff}-${_realname}")
 conflicts=("${_mingw_suff}-${_realname}")
-pkgdesc="MinGW-w64 headers for cross-compiler"
-pkgver=5.0.0.4834.749e8116
+pkgver=5.0.0.4838.40dcdd35
 pkgrel=1
+pkgdesc="MinGW-w64 winpthreads library for cross-compiler"
```
Comment (Alexpux) [inline mingw-w64-cross-tools-git/PKGBUILD]: Also why it here

**Comment by Alexpux** [comment]
You place winpthreads PKGBUILD instead TOOLS. Revert it

**Comment by JPeterMugaas** [comment]
Sometimes going haywire.  I have to chuck this.

**Pull request closed without merging**